workflow/rocoto/gefs_tasks.py

```diff
@@ -159,29 +159,29 @@ def efcs(self):
 
         return task
 
-    def atmprod(self):
+    def atmosprod(self):
         atm_master_path = self._template_to_rocoto_cycstring(self._base["COM_ATMOS_MASTER_TMPL"], {'MEMDIR': 'mem#member#'})
         deps = []
         data = f'{atm_master_path}/{self.cdump}.t@Hz.master.grb2f#fhr#'
         dep_dict = {'type': 'data', 'data': data, 'age': 120}
         deps.append(rocoto.add_dependency(dep_dict))
         dependencies = rocoto.create_dependency(dep=deps)
 
-        atm_prod_envars = self.envars.copy()
+        atmosprod_envars = self.envars.copy()
         postenvar_dict = {'ENSMEM': '#member#',
                           'MEMDIR': 'mem#member#',
                           'FHRLST': '#fhr#',
                           }
         for key, value in postenvar_dict.items():
-            atm_prod_envars.append(rocoto.create_envar(name=key, value=str(value)))
+            atmosprod_envars.append(rocoto.create_envar(name=key, value=str(value)))
 
         resources = self.get_resource('atmos_products')
 
-        task_name = f'atm_prod_mem#member#_f#fhr#'
+        task_name = f'atmosprod_mem#member#_f#fhr#'
```

Comment:
Trying to keep GEFS tasks more readable (with the intention of eventually going through GFS and do the same).
```suggestion
        task_name = f'atmos_prod_mem#member#_f#fhr#'
```